app/components/wallet/transactions/Transaction.scss

```diff
@@ -241,10 +241,6 @@
     letter-spacing: 1px;
   }
   // ========= DETAILS =========
-  .content {
```

Comment:
I think the idea was to remove only the last line? If there is more than one, there should be lines in between.

![trx_middle_line](https://user-images.githubusercontent.com/1937074/60374828-4dddaa80-99d4-11e9-9ef4-8306de043211.png)
(just remove the last one)

Reply:
@leivaburto yeah. @YankoFelipe  the idea it was to only remove the last line